custom/icds_reports/tasks.py

```diff
@@ -1176,3 +1176,35 @@ def _bust_awc_cache():
     for key in reach_keys:
         cache.delete(key)
     create_datadog_event('redis: delete dashboard keys', 'finish')
+
+
+@task(queue='dashboard_comparison_queue')
+def run_citus_experiment_raw_sql(sql, data_source="Unknown"):
+    experiment_context = {
+        "data_source": data_source,
+        "sql_query": sql,
+    }
+    experiment = DashboardQueryExperiment(name="Dashboard Query Experiment", context=experiment_context)
+    with experiment.control() as control:
+        db_alias = get_icds_ucr_db_alias()
+        with connections[db_alias].cursor() as cursor:
+            cursor.execute(sql)
+            control.record(_dictfetchall(cursor))
+
+    with experiment.candidate() as candidate:
+        db_alias = 'citus'
```

Comment:
this needs to be changed once we have the final db alias in prod

Reply:
Perhaps wort adding adding a `# todo` in the code itself.